ikvmc/remapper.cs

```diff
@@ -1549,6 +1549,9 @@ internal void Emit(ClassLoaderWrapper loader, CodeEmitter ilgen)
 			// HACK if the class name contains a comma, we assume it is a .NET type
 			if(Class.IndexOf(',') >= 0)
 			{
+#if NETSTANDARD
```

Comment:
netcore aswell?

Reply:
As I see it, NETSTANDARD is defined when building coreapp targets.